### ISSUE TITLE
Backticks in examples don't work, see e.g. bottom of https://dwc.tdwg.org/list/2020-08-20.html (also on current site)

### ISSUE THREAD

**Issue by peterdesmet** [tdwg/dwc#428]

**Comment by gkampmeier** [comment]
Are these backticks needed for anything (they are not single quote marks, with which someone may have confused them)? Given that these are used in examples, any sort of single or double quote mark would seem implied and more confusing to the user (i.e., should they be in my data for some reason?).

**Comment by peterdesmet** [comment]
The backticks are markdown for `<code></code>`. We use them for literal values (like examples, variables) and are visually distinct from surrounding text. They have the benefit over quotes that it isn’t confusing what should be included _and_ that it is possible to include quotes. Like so: `value: "quoted string"`.

**Comment by peterdesmet** [comment]
Note that this is resolved on the [latest version](https://dwc.tdwg.org/list/), but not on older versions like https://dwc.tdwg.org/list/2020-08-20.html. But I assume that would require the script to be run again, pulling in recent changes, which we don't want.

**Comment by tucotuco** [comment]
I think it is fine to leave outdated versions as they are.

**Comment by baskaufs** [comment]
I agree. They are only for historical purposes and for many of the terms, the whole reason they have a new version is because of this change. So if we change the older versions, the reason for the change will be lost.
